cooltools/api/snipping.py

```diff
@@ -351,19 +344,23 @@ def select(self, region1, region2):
             region1_coords, region2_coords
```

Comment:
could you please please leave a comment here that matrix is fetched sparse here ?